.github/tests-visual/visual-record.js

```diff
@@ -127,7 +127,9 @@ async function main() {
         'modules_modules.png': START_URL + '?page=modules/modules',
         'modules_actions.png': START_URL + '?page=modules/actions',
         'users_users.png': START_URL + '?page=users/users',
+        'users_edit.png': START_URL + '?page=users/users&user_id=1',
         'users_roles.png': START_URL + '?page=users/roles',
+        'users_role_add.png': START_URL + '?page=users/roles&func=add&default_value=1',
```

Comment:
in der einfachen demo haben wir keine rollen, daher arbeiten wir hier mit "role-add" statt edit einer vorhandenen rolle